pom.xml

```diff
@@ -1771,5 +1772,75 @@
                 </plugins>
             </build>
         </profile>
+
+
+
+        <!-- JaCoCo test coverage. Will set ${surefire.jacoco.args} to be used in Arquillian config and as surefire jvm argument. -->
```

Comment:
There are no arquillian tests in WildFly core
And I don't know if jacoco will work with our WildFly test runner.

Question is should we even have this enabled for testsuite at all, maybe having it for unit tests would be enough

Reply:
There are no arquillian tests in WildFly core, you are right, I update this comment.

Yes, JaCoCo is basically java agent, it works correctly also with WildFly Test Runner.

If we want to know test coverage of this testsuite, we need to enable JaCoCo in this testsuite. If we enable JaCoCo only in unit tests, we don't have test coverage data for testsuite. Testsuite adds more test coverage then unit tests.

Anyway, JaCoCo is disabled by default in this PR, it needs to be activated by -Dcoverage.